.github/workflows/gen_vc.yml

```diff
@@ -0,0 +1,38 @@
+name: VC gen
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-vc:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build V
+      run: make
+    - name: Regenerate v.c and v_win.c
+      run: |
+        COMMIT_HASH=$(git rev-parse --short HEAD)
+        COMMIT_MSG=$(git log -1 --oneline --pretty='%s' HEAD)
+
+        rm -rf vc
+        git clone https://github.com/vlang/vc
```

Comment:
it would be slightly faster, if you add `--depth=1` here

Reply:
not sure if it will allow pushing after that though 🤔

Reply:
in any case, we can test later

Reply:
I don't think you can push to a repo after a shallow clone, but I may be wrong.